src/content/docs/crowdin/translation-process/crowdin-ai.mdx

```diff
@@ -204,22 +207,58 @@ To disable the AI provider, follow these steps:
   2. In the **Providers** tab, click <Icon name="mdi:dots-horizontal" class="inline-icon" /> toward the needed provider and select **Disable**. Alternatively, double-click on the needed provider and clear **Enabled** in the provider's settings.
 </Steps>
 
-### Topping Up Account Balance
+### Adding Funds to Account Balance
 
-MT engines and AI models managed by Crowdin are separately paid services. To use them, you first need to top up your Crowdin account balance.
+MT engines and AI models managed by Crowdin are separately paid services. To use them, you first need to add funds to your Crowdin account balance.
 
 <ReadMore>
-  Read more about [Topping Up Account Balance](/crowdin-managed-services/#topping-up-account-balance).
+  Read more about [Adding Funds to Account Balance](/crowdin-managed-services/#adding-funds-to-account-balance).
 </ReadMore>
 
 ### AI Settings for the Account
 
-To configure default settings for AI prompts across your Crowdin account, open your profile and go to **AI > Settings**.
+To configure default AI settings across your Crowdin account, open your profile and go to **AI > Settings**.
 
-**Prompt to be used in editor** &ndash; select the default AI prompt that will be used in the Editor for all projects in your Crowdin account. If needed, this default setting can be overridden for each project individually in the project's **Settings > AI > Settings**.
+These default settings can be overridden for each project individually in the project's **Settings > AI > Settings** as needed.
 
 <Image src={aiSettings} alt="AI Settings" />
 
+* **AI Assistant in editor** &ndash; select the default prompt for interactions with AI Assistant in the Editor across all projects in your Crowdin account.
+
+* **AI Suggestion in editor** &ndash; select the default prompt for AI suggestions that will appear in the Editor across all projects in your Crowdin account. If no account-wide setting is configured and no prompt is manually selected for a specific project, AI suggestions will not be displayed in the Editor for that project. Once a prompt is selected, Crowdin will use it to generate and display translation suggestions in the Editor.
+
+<Aside>
```

Comment:
Add caution type and add indentation to this Aside. Currently, without the indentation, it breaks the list.

Reply:
Added caution type. Usually, we use caution with titles. Should I add any title as well?

Since I'm creating separate headings for these sections (AI Assistant in editor, AI Suggestion in editor, Shortcuts for chat with AI in editor, etc.), I think the indentation for this Aside is not necessary in this case.

Reply:
No, it's not necessary.